garak/generators/huggingface.py

```diff
@@ -79,6 +79,9 @@ def _load_client(self):
             set_seed(_config.run.seed)
 
         pipeline_kwargs = self._gather_hf_params(hf_constructor=pipeline)
+        pipeline_kwargs["truncation"] = (
```

Comment:
Sometimes I wish this were just the default for HF but c'est la vie. We may want to consider whether `truncation = True` requires `max_len` to be set -- I've had HF yell at me for not specifying both but it may be a corner case that I encountered.